oak/src/OnlineTicTacToe/MessageHandler.ts

```diff
@@ -0,0 +1,36 @@
+import { ConnectedGames } from "./connectedGames.ts";
+import { GameName,GameEvents,GameData,OnlineGame,WebSocketEvent } from "./OnlineGame.ts";
+
+type MessageHandler = (this: WebSocket, ev: MessageEvent<any>) => any
+
+export function onMessageGameHandler(
+  gameName: GameName,
+  connectedGames: ConnectedGames,
+): MessageHandler {
+  
+  return (msg: MessageEvent) => {
+    const data = JSON.parse(msg.data)
+
+    switch (data.event) {
```

Comment:
A switch case with no default case is bad

Reply:
yea I'll use if's instead

```suggestion
    if (data.event == GameEvents.updateGame) {
```